extensions/wikia/AdEngine/js/lookup/prebid/prebid.js

```diff
@@ -105,13 +105,36 @@ define('ext.wikia.adEngine.lookup.prebid', [
 	}
 
 	function getSlotParams(slotName) {
-		var params;
+		var slotParams;
+
+		if (win.pbjs && typeof win.pbjs.getBidResponses === 'function') {
+			var params = win.pbjs.getBidResponses(slotName) || {};
+
+			if (params && params[slotName] && params[slotName].bids && params[slotName].bids.length) {
+				var targeting,
+					priority = adaptersRegistry.getPriority();
+
+				params[slotName].bids.forEach(function (param) {
+					if (!targeting) {
+						targeting = param;
```

Comment:
I'm not sure if this object is equal to slot targeting that we push to DFP.

Reply:
BTW: is it targeting or `bidParams`?

Reply:
1) We take exactly the same params of winner after loop:
`slotParams = targeting.adserverTargeting;`
2) You right, I can rename that.